### PR TITLE
Add Serializable object

### PR DESCRIPTION
**Pull request by pehala**
* Reworked and move it out of the chat app.

### CODE REVIEW

File: utils/serializable.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"""Module containing serializable helper class"""
+
+
+# pylint: disable=too-few-public-methods
+class Serializable:
+    """Simple serializable class that exports all field name into a dict ignoring protected fields"""
+    def to_dict(self):
+        """Serialize object into dict, it ignores all protected attributes"""
+        return {key: value for key, value in vars(self).items() if not key.startswith("_")}
```
Comment (Sanquira) [inline utils/serializable.py:9]: How about nested lists/dicts/whatever-hotdog?

Comment (pehala) [reply utils/serializable.py:9]: That situation cannot happen, `vars` (and by extension `__dict__`) returns simple dict like that {"field_name": field_value}
https://docs.python.org/3/library/functions.html#vars

Comment (Sanquira) [reply utils/serializable.py:9]: What if field_value is object?

Comment (pehala) [reply utils/serializable.py:9]: Why should that matter?

Comment (pehala) [reply utils/serializable.py:12]: Docs should be better now

File: utils/serializable.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+"""Module containing serializable helper class"""
+
+
+# pylint: disable=too-few-public-methods
+class Serializable:
+    """Simple serializable class that exports all field name into a dict ignoring protected fields"""
+    def to_dict(self):
+        """
+        Returns all object fields in a dict form to be used for serialization purposes.
+        It ignores all protected attributes and methods
```
Comment (Sanquira) [inline utils/serializable.py:10]: Missing '.'

Comment (pehala) [reply utils/serializable.py]: Solved

**Pull request merged**